imap_processing/tests/swe/test_swe_l2.py

```diff
@@ -109,6 +111,201 @@ def test_calculate_flux():
     assert type(flux) == np.ndarray
 
 
+def test_find_angle_bin_indices():
+    """Test find_angle_bin_indices function."""
+    spin_angle_bin_center = np.arange(6, 360, 12)
+    spin_angle_bin_edges = np.arange(0, 360, 12)
+
+    start_angles = [0, 12, 24, 36, 48, 60, 72, 84, 96, 108]
```

Comment:
This is definitely above and beyond what I was expecting! :) I'd say you really only need to test a few intermediate values and the extremes, and I'd generally put them all into a single array at once. `np.array([0, 1, 6, 9, 12, 355])`